src/plugins/auth.js

```diff
@@ -0,0 +1,45 @@
+import Jwt from '@hapi/jwt'
+
+import { config } from '../config/index.js'
+
+/**
+ * @satisfies {ServerRegisterPlugin}
+ */
+export const auth = {
+  plugin: {
+    name: 'auth',
+    async register(server) {
+      await server.register(Jwt)
+
+      server.auth.strategy('jwt_token', 'jwt', {
+        keys: {
+          uri: config.get('oidcJwksUri')
+        },
+        verify: {
+          aud: config.get('oidcVerifyAud'),
+          iss: config.get('oidcVerifyIss'),
+          sub: false,
+          nbf: true,
+          exp: true
+        },
+        /**
+         * @param {Artifacts} artifacts
+         */
+        validate: (artifacts) => {
```

Comment:
perhaps we should do a group check here? This would prevent someone who's not authorised from accessing the service. Bonus points for a similar scope support to the forms-designer but not a blocker if it's just a simple group check.

```
if (artifacts.decoded.payload.groups does not contain the group ID) {
     return {
         isValid: false
     }
}
```